crates/bevy_audio/src/volume_level.rs

```diff
@@ -0,0 +1,188 @@
+/// Represents a volume level value.
+#[derive(Clone, Copy, Debug)]
+pub enum VolumeLevel {
+    /// Volume Amplitude Ratio (unitless)
+    ///
+    /// # Notes
+    ///
+    /// `Self::Amplitude(a)` is defined for all `f32` values. However, negative values represent a
+    /// phase inversion. If `a` and `b` have the same magnitude (`a.abs() == b.abs()`), but opposite
+    /// sign (`a == -b`), then `Self::Amplitude(a) == Self::Amplitude(b)`, as this phase information
+    /// is currently ignored. As such, you should use positive values for this variant to avoid confusion.
+    Amplitude(f32),
```

Comment:
At the risk of some bikeshedding (and probably pedantry), "amplitude" refers to the volume itself, not a specific "unit" -- `Linear` would be a better term, as it is the term that is most used for the unitless gain ratio (this applies to the method name line 26 too). I can imagine some users being confused when looking at this variant by itself and not knowing whether we are talking about the linear gain or the logarithmic one (in dB), and having to look up the enum definition to clear the confusion.